server/Arcadia.Assistant.Calendar.Notifications/EventAssignedToApproverNotificationActor.cs

```diff
@@ -0,0 +1,125 @@
+﻿namespace Arcadia.Assistant.Calendar.Notifications
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+
+    using Akka.Actor;
+    using Akka.Event;
+
+    using Arcadia.Assistant.Calendar.Abstractions;
+    using Arcadia.Assistant.Calendar.Abstractions.EventBus;
+    using Arcadia.Assistant.Configuration.Configuration;
+    using Arcadia.Assistant.Notifications;
+    using Arcadia.Assistant.Notifications.Email;
+    using Arcadia.Assistant.Organization.Abstractions;
+    using Arcadia.Assistant.Organization.Abstractions.OrganizationRequests;
+    using Arcadia.Assistant.UserPreferences;
+
+    public class EventAssignedToApproverNotificationActor : UntypedActor, ILogReceive
```

Comment:
that's the thing, it's not really a general notification actor. It has to know all about email/push settings which is not really good. We either need to rename this one, or split it on two (push and emails) with one more layer between them

Reply:
We already have common email and common push actors, we don't need more layers.
We can split this on `EventAssignedToApproverEmailNotificationActor` and `EventAssignedToApproverPushNotificationActor` and each actor responsibility will be to wrap and send notification of specific type.
Or we can assume that this concrete actor responsibility is to send notifications of all known types about event assigned to approver.
First is more verbose, but each actor has only one reason to change. Second is less verbose, but actor has several reasons to change. What do you prefer?

Reply:
Done.